doc/library/gpuarray/dnn.txt

```diff
@@ -1,7 +1,7 @@
 .. _libdoc_gpuarray_dnn:
 
 ===========================================
-:mod:`theano.sandbox.gpuarray.dnn` -- cuDNN
+:mod:`gpuarray.dnn` -- cuDNN
```

Comment:
I know they do, but it makes the internal links not work sometimes because sphinx thinks the module is `gpuarray` not `theano.gpuarray`.